components/organisms/Breadcrumb.js

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { faAngleRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Link from "next/link";
+
+import { useContext } from "react";
+import { LanguageContext } from "../../context/languageProvider";
+import en from "../../locales/en";
+import fr from "../../locales/fr";
+
+/**
+ * breadcrumb component
+ */
+
+const convertBreadcrumb = (string) => {
+  return string
+    .replace(/-/g, " ")
+    .replace(/oe/g, "ö")
+    .replace(/ae/g, "ä")
+    .replace(/ue/g, "ü");
+};
+
+export default function BreadcrumbsReact(props) {
+  const router = useRouter();
+  const [breadcrumbs, setBreadcrumbs] = useState(null);
+
+  useEffect(() => {
+    if (router) {
+      const linkPath = router.asPath.split("/");
+      linkPath.shift();
+      const pathArray = linkPath.map((path, i) => {
+        return {
+          breadcrumb: path,
+          href: "/" + linkPath.slice(0, i + 1).join("/"),
+        };
+      });
+      setBreadcrumbs(pathArray);
+    }
+  }, [router]);
+  if (!breadcrumbs) {
+    return null;
+  }
+
+  const { items } = useContext(LanguageContext);
+  const language = items.language;
+  const t = language === "en" ? en : fr;
+  return (
+    <nav aria-label="breadcrumbs">
+      <div>
+        <div className="inline">
+          <Link href="/">
+            <a className="inline">{t.home}</a>
+          </Link>
+        </div>
+        {breadcrumbs.map((breadcrumb, i) => {
+          return (
+            <div key={breadcrumb.href} className="inline">
+              <Link href={breadcrumb.href}>
+                <a>
+                  {"  "}
+                  {breadcrumb.breadcrumb && (
+                    <FontAwesomeIcon icon={faAngleRight} color="#2B4380" />
```

Comment:
i think you are missing size here  size="sm"

Reply:
possibly.... it doesn't appear to make a difference (except it might actually make it too small).  i''ll play around with it a bit more